src/fu-util.c

```diff
@@ -2525,6 +2591,13 @@ main (int argc, char *argv[])
 		     /* TRANSLATORS: sets something in daemon.conf */
 		     _("Modifies a daemon configuration value."),
 		     fu_util_modify_config);
+	fu_util_cmd_array_add (cmd_array,
+		     "reinstall",
+		     "[DEVICE-ID]",
+		     /* TRANSLATORS: command description */
+		     _("Re-install current firmware on the device."),
```

Comment:
Is't _reinstall_ one word? If it's not, ignore me.

Reply:
It is. I think you're right.

Reply:
Oh - elsewhere in the code it's also "re-install" (--allow-reinstall help).  I'll fixup both.

Reply:
Great, thanks.